ui/apps/platform/src/Containers/Vulnerabilities/VulnerablityReporting/components/ReportParametersDetails.tsx

```diff
@@ -53,20 +53,17 @@ function ReportParametersDetails({
             </DescriptionListDescription>
         );
     }
-    /*
-    // Ross CISA KEV includeKnownExploit?
     if (
         isFeatureFlagEnabled('ROX_SCANNER_V4') &&
-        isFeatureFlagEnabled('ROX_WHATEVER') &&
+        isFeatureFlagEnabled('ROX_CISA_KEV') &&
```

Comment:
Is this a different feature flag than `'ROX_KEV_EXPLOIT'` in backend contribution?

Reply:
I ended up changing things, including the feature flag name. Now it's `ROX_CISA_KEV` - https://github.com/stackrox/stackrox/pull/16868